portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+<title>MyPath</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyPath</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>SIMULATION_MENU</td>
+	<td>//div[contains(@class,'control-menu')]//button[@data-qa-id='simulation']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_BUTTON</td>
+	<td>//div[contains(@class,'row size-selector')]//button[child::span[text()='Custom']]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_HEIGHT</td>
+	<td>//div[contains(@id,'customSizeSelector')]//input[contains(@id,'height')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_WIDTH</td>
+	<td>//div[contains(@id,'customSizeSelector')]//input[contains(@id,'width')]</td>
+	<td></td>
```

Comment:
Hey, here you can see that the only difference between the xpaths is the id of the input, correct? 
What you did is right, but if you want to not repeat yourself what you can do is create only one xpath for these two cases and put a variable that can assume one of the values you are looking for: the height or the width. Like this:

`//div[contains(@id,'customSizeSelector')]//input[contains(@id,'${nameOfTheVariable}')]`

Tell me if this works, thank you!

btw If you choose to create only one xpath, you will need to change the name of the XPath, you can put CUSTOM_SIZE_INPUT or any other name of your choice.

Reply:
That's a good idea! Thanks for the advice. I tested it and it worked nicely. Thank you!